.circleci/config.yml

```diff
@@ -22,6 +22,7 @@ jobs:
       - run:
           name: Install nteract desktop
           command: |
+            sudo apt install libgif-dev
```

Comment:
Interesting.

Reply:
Yeah without this, we were getting a node-gyp error with canvas. It didn't actually break the build, but you know, can't resist getting rid of errors 😄. [Here](https://circleci.com/gh/nteract/desktop-integration-tests/13) is an example of the log with the error.